packages/altive_lints/lint_test/lints/avoid_single_child.dart

```diff
@@ -44,6 +44,12 @@ class MyWidget extends StatelessWidget {
             Text('World'),
           ],
         ),
+        ListView(
+          children: List.generate(
+            10,
+            (index) => Text('Hello $index'),
+          ),
+        ),
```

Comment:
As shown here, a cast error would occur when the value of `children` is not a `ListLiteral` (`[]`).